### PR TITLE
feat: clearable prop

### PR DESCRIPTION
**Pull request by schelmo**
i added a jest test for it, do you want a cypress test too?

### CODE REVIEW

File: docs/api-reference.md [hunk 1]
```diff
@@ -218,6 +218,16 @@
...
 
+### clearable
+
+- **Type**: `boolean`
+
+- **Default**: `true`
+
+- **Details**:
+
+  Wether to allow clearing the selected options
```
Comment (iendeavor) [inline docs/api-reference.md:229]:
```suggestion
  Whether to allow clearing the selected options
```

File: docs/api-reference.md [hunk 2]
```diff
@@ -218,6 +218,16 @@
 
   Whether to collapse dropdown after selected/removed.
 
+### clearable
+
```
Comment (iendeavor) [inline docs/api-reference.md:222]:
```suggestion

New in `v2.9.0+`

```

**Review by iendeavor — COMMENTED** [review]
I think this feature is duplicated with the `min` prop.

**Comment by schelmo** [comment]
ah sure. completely missed the min value ;)

**Pull request closed without merging**